certgen/certProcessor/src/main/java/org/incredible/FileUploader.java

```diff
@@ -0,0 +1,15 @@
+package org.incredible;
+
+import org.incredible.certProcessor.store.ICertStore;
+
+import java.io.File;
+import java.io.IOException;
+
+public class FileUploader {
```

Comment:
I see this isn't helping much being static class. We have a storage uploader under utils. Reuse and throw away this.

Reply:
Removed